test/gen_rpc/multi_rpc_functional_SUITE.erl

```diff
@@ -0,0 +1,371 @@
+%%% -*-mode:erlang;coding:utf-8;tab-width:4;c-basic-offset:4;indent-tabs-mode:()-*-
+%%% ex: set ft=erlang fenc=utf-8 sts=4 ts=4 sw=4 et:
+%%%
+%%% Copyright 2015 Panagiotis Papadomitsos. All Rights Reserved.
+%%%
+
+-module(multi_rpc_functional_SUITE).
+-author("Panagiotis Papadomitsos <pj@ezgr.net>").
+
+%%% CT Macros
+-include_lib("test/gen_rpc/include/ct.hrl").
+
+%%% Common Test callbacks
+-export([all/0, init_per_suite/1, end_per_suite/1, init_per_testcase/2, end_per_testcase/2]).
+
+%%% Testing functions
+-export([supervisor_black_box/1
```

Comment:
In the interest of keeping the same coding style, can you move the commas on the upper line like in all other modules?